addons/spreadsheet/static/src/chart/odoo_chart/odoo_chart.js

```diff
@@ -133,3 +133,38 @@ export class OdooChart extends AbstractChart {
         }
     }
 }
+
+export function getEventCallback(getters, datasets, labels, res_model) {
+    const actionService = getters.getActionService();
+    return {
+        onClick : async (event, items) => {
+            if (!items.length) {
+                return;
+            }
+            const {datasetIndex, index} = items[0];
+            const dataset = datasets[datasetIndex];
+            let name = labels[index];
+            if (dataset.label) {
+                name += ` / ${dataset.label}`;
+            }
+            await actionService.doAction(
+                {
+                    name,
+                    type: "ir.actions.act_window",
+                    res_model,
+                    views: [[false, 'list']],
```

Comment:
It would be even better to preserve the original action to be sure we land with the correct list view.
you can add it to the definition in `spreadsheet_edition/[...]/graph_view.js` and `spreadsheet_edition/[...]/odoo_chart_insertion.js`.

Then use `navigateTo`

Otherwise, if you insert a chart of time sheets, you might land on the analytic account list view (they are the same model) (or the other way around, depending on which view is first)

We did it for "See record(s)" of lists and pivots.
https://github.com/odoo/enterprise/pull/54798/files#diff-ebe61bfd412de8c277d6906b6c6e45f50e6db50903104fa045a5af58f5a2188fR71